controller/response/responder.go

```diff
@@ -150,6 +160,15 @@ func (rr *RequestResponderImpl) RespondWithNotFound() {
 	})
 }
 
+func (rr *RequestResponderImpl) RespondWithRecordNotFound(e error) {
```

Comment:
I do not see the need for both `RespondWithNotFound` and `RespondWithRecordNotFound`. If the difference is simply the `e error` used as `cause`. I'd just make `RespondWithNotFoundWithCause(e error)` and `RespondWIthNotFound()` and have one call the other.

The word "record" doesn't have any significant meaning. URLs may contain non-record accounts for virtual objects that do not have records. So I feel it muddies the waters.

Reply:
Makes sense. Fixed